### PR TITLE
Fixing remaining days on phase (issue 30)

### LINKED ISSUE #130: Cálculo de dias restantes da fase está errado

### PR DESCRIPTION
**Pull request by erickzanardo**
This PR closes #130

### How was it before?

- the application was calculating the remaining days wrong, it was using the phase's final date and initial date

### What has changed?

- the bug was fixed, now the application uses the correct date, the phase's final date and today's date

### What should I pay attention when reviewing this PR?

Nothing special

### Is this PR dangerous?

no

### CODE REVIEW

File: app/models/phase.rb [hunk 1]
```diff
@@ -84,7 +84,7 @@ def current_status
   end
 
   def remaining_days
-    (final_date.to_date - initial_date.to_date).to_i
+    (final_date.to_date - Date.today).to_i
```
Comment (guimello) [inline app/models/phase.rb]: I guess you should clamp the negative results.

```ruby
[0, (final_date.to_date - Date.today).to_i].max
```

**Review by guimello — CHANGES_REQUESTED** [review]

File: spec/models/phase_spec.rb [hunk 2]
```diff
@@ -211,5 +211,10 @@
     subject { phase.remaining_days }
 
     it { is_expected.to be 2 }
+
+    context "when the phase has long been finished" do
```
Comment (guimello) [inline spec/models/phase_spec.rb:215]: Could you please add a test for negative days?

**Review by guimello — APPROVED** [review]
after the test, just 🚀
